odoo/addons/base/models/res_country.py

```diff
@@ -106,7 +111,10 @@ def create(self, vals_list):
     def write(self, vals):
         if vals.get('code'):
             vals['code'] = vals['code'].upper()
-        return super(Country, self).write(vals)
+        res = super(Country, self).write(vals)
```

Comment:
while you're touching this line, you can modernize it ;)
```suggestion
        res = super().write(vals)
```